oracle/OracleBillDownload.py

```diff
@@ -0,0 +1,81 @@
+import oci
+import os
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import pytz
+import json
+
+# This script downloads all of the cost, usage, (or both) reports for a tenancy (specified in the config file).
```

Comment:
Very minor: in python, if you move those comments at the beginning of your script as a multi-line string, eg:
```
'''
This script downloads all of the cost [...]

Pre-requisites: [...]
'''
import oci
[...]
```

it is automatically put into the `__doc__` built-in variable. See https://github.com/flexera/optima-bill/blob/master/examples/python3/bill_upload_example.py for an example.